data/cmdvartab

```diff
@@ -106,14 +117,31 @@ VARDESC battery.charger.status "Battery charger status"
 
 VARDESC ambient.temperature "Ambient temperature (degrees C)"
 VARDESC ambient.temperature.alarm "Ambient temperature alarm is active"
+VARDESC ambient.temperature.status "Ambient temperature status relative to the configured thresholds"
 VARDESC ambient.temperature.alarm.maximum "Maximum allowed ambient temperature"
 VARDESC ambient.temperature.alarm.minimum "Minimum allowed ambient temperature"
 VARDESC ambient.temperature.alarm.enable "Enable ambient temperature alarm"
+VARDESC ambient.temperature.low "Temperature threshold low"
+VARDESC ambient.temperature.low.warning "Temperature threshold low warning"
+VARDESC ambient.temperature.low.critical "Temperature threshold low critical"
+VARDESC ambient.temperature.high "Temperature threshold high"
+VARDESC ambient.temperature.high.warning "Temperature threshold high warning"
+VARDESC ambient.temperature.high.critical "Temperature threshold high critical"
 VARDESC ambient.humidity "Ambient humidity"
 VARDESC ambient.humidity.alarm "Ambient humidity alarm is active"
+VARDESC ambient.humidity.status "Ambient humidity status relative to the configured thresholds"
 VARDESC ambient.humidity.alarm.maximum "Maximum allowed ambient humidity"
 VARDESC ambient.humidity.alarm.minimum "Minimum allowed ambient humidity"
 VARDESC ambient.humidity.alarm.enable "Enable ambient humidity alarm"
+VARDESC ambient.humidity.low "Ambient humidity threshold low"
+VARDESC ambient.humidity.low.warning "Ambient humidity threshold low warning"
```

Comment:
Is there a common unit of measurement here (e.g. "%hum"?) that can be appended to relevant lines?